lib/material_chips/material_chip.dart

```diff
@@ -72,7 +61,7 @@ class MaterialChipComponent extends RootFocusable implements HasRenderer {
   @Input()
   bool hasLeftIcon = false;
 
-  /// An [ItemRenderer] function (takes an object and returns a string).
+  /// An `ItemRenderer` function (takes an object and returns a string).
```

Comment:
(Curious why you made this change from type reference to code formatting.)

Reply:
I suspect this is because the dart doc linking syntax didn't have any special formatting in the gallery. I updated the gallery so that it will display like a code block instead until we find a good solution to dartdoc links.